starter_kits/Ruby/hlt/commands.rb

```diff
@@ -0,0 +1,12 @@
+# All viable commands that can be sent to the engine
```

Comment:
This is not technically correct, since now you have cardinals there as well, they are not commands per-se. But I can live with it.